api/shape/Curve.js

```diff
@@ -8,17 +8,24 @@ export const Curve = Shape.registerMethod(
   'Curve',
```

Comment:
What do curves mean?

Reply:
It produces a curved path that passes smoothly through the control points.